core/src/main/scala/cats/syntax/monad.scala

```diff
@@ -20,3 +24,28 @@ final class MonadIdOps[A](private val a: A) extends AnyVal {
    */
   def iterateUntilM[F[_]](f: A => F[A])(p: A => Boolean)(implicit M: Monad[F]): F[A] = M.iterateUntilM(a)(f)(p)
 }
+
+final class MonadBooleanOps[F[_]](private val fa: F[Boolean]) extends AnyVal {
+
```

Comment:
She we also add nand, nor, any, all?

Reply:
Since it seems that `and` and `or` are already niche enough that most folks seem to be against this addition, I doubt there is value on those others.

IMHO, these two are useful enough to be on **cats** but that is up to the consensus of everyone.